src/Data/Vec/Relation/Binary/Equality/Setoid.agda

```diff
@@ -68,6 +68,11 @@ open PW public using (++⁺ ; ++⁻ ; ++ˡ⁻; ++ʳ⁻)
 ++-identityʳ []       = []
 ++-identityʳ (x ∷ xs) = refl ∷ ++-identityʳ xs
 
+++-assoc : ∀ {n m k} (xs : Vec A n) → (ys : Vec A m) 
+    → (zs : Vec A k) → (xs ++ ys) ++ zs ≋ xs ++ (ys ++ zs)
```

Comment:
Minor formatting comments:
- You don't need the arrows between named quantified variables so you can write `{n m k} (xs : Vec A n) (ys : Vec A m)  (zs : Vec A k)` on one line.
- According to the style guide arrows go at the end of lines.
- Could you align the `xs` and `zs` in the proof below? Helps with readability!